pkg/kubelet/nodeshutdown/systemd/inhibit_linux.go

```diff
@@ -138,7 +149,11 @@ func (bus *DBusCon) MonitorShutdown() (<-chan bool, error) {
 
 	go func() {
 		for {
-			event := <-busChan
+			event, ok := <-busChan
+			if !ok {
+				close(shutdownChan)
```

Comment:
when dbus will restart is it expected that busChan will return ok=false?

Reply:
yes